src/test/java/tr/com/srdc/cda2fhir/testutil/ClinicalDocumentMetadataGenerator.java

```diff
@@ -0,0 +1,110 @@
+package tr.com.srdc.cda2fhir.testutil;
+
+import org.hl7.fhir.utilities.xhtml.HierarchicalTableGenerator.Title;
+import org.openhealthtools.mdht.uml.cda.ClinicalDocument;
+import org.openhealthtools.mdht.uml.hl7.datatypes.CE;
+import org.openhealthtools.mdht.uml.hl7.datatypes.CS;
+import org.openhealthtools.mdht.uml.hl7.datatypes.II;
+import org.openhealthtools.mdht.uml.hl7.datatypes.INT;
+import org.openhealthtools.mdht.uml.hl7.datatypes.ST;
+import org.openhealthtools.mdht.uml.hl7.datatypes.TS;
+
+public class ClinicalDocumentMetadataGenerator {
+	ClinicalDocument doc;
+	static final public String DEFAULT_REALM_CODE = "US";
+	static final public String DEFAULT_ID_ROOT = "1.2.840.114350.1.13.88.3.7.8.688883.41197285";
+	static final public String DEFAULT_ASSN_AUTH = "EPC";
```

Comment:
yeah change some of this, it's epic specific.  the assigning authority, ID_ROOT, SET_ID_EXT, and ID_ROOT. You can make them really anything, but these are pulled from an Epic example so we should just generalize them since it's just for this testing thing.  You can also set the date to Jan 1 2019.

Reply:
Updated. LMK what you think.